docs/Scaling/Kubernetes/service-mesh.md

```diff
@@ -0,0 +1,69 @@
+# Enabling KUMA Service Mesh (Optional)
+
+For enhancing security in the Kubernetes deployment, we use KUMA Service Mesh to enable mTLS between services. To learn more about KUMA visit their [documentation](https://kuma.io/docs/)
```

Comment:
You can drop the "we".  You can say "use KUMA Service Mesh" if you mean the user should employ KUMA. Or you can say the toolkit uses to KUMA to accomplish this enhanced security.